packages/SwingSet/package.json

```diff
@@ -52,6 +52,7 @@
     "@endo/promise-kit": "^0.2.56",
     "@endo/ses-ava": "^0.2.40",
     "@endo/zip": "^0.2.31",
+    "better-sqlite3": "^8.2.0",
```

Comment:
can we make this a devDependency? I feel like I've managed this in `misc-tools/` scripts before (like extract-transcript?).

Reply:
oh, maybe the fact that it's in `tools/` is why it needs to be a real `dependency`